src/main/kotlin/uk/gov/justice/digital/hmpps/courthearingeventreceiver/model/Offence.kt

```diff
@@ -26,6 +26,9 @@ data class Offence(
   val wording: String,
 
   @JsonProperty("offenceLegislation")
-  val offenceLegislation: String? = null
+  val offenceLegislation: String? = null,
+
+  @JsonProperty("listingNumber")
+  val listingNumber: Number? = null
```

Comment:
Should probably use `Int` rather than `Number` to be more explicit. Not sure what the semantics of this would be if you left it open as any Number class. Could even use a UInt. Wish I'd thought to do that elsewhere now but better late than never!